### PR TITLE
Add three more icons (adjust, bookmark-regular, bookmark-solid)

### LINKED ISSUE #105: New icons for CC Search browser extension
## Description
We need to add three new icons for the CC Search browser extension. The icons can be found in fontawesome and are:
1. Adjust
2. Bookmark regular
3. Bookmark solid

Screen Shot 2020-07-01 at 12 56 52

The [Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=0%3A135&viewport=-1282%2C-26%2C2.5631163120269775) is updated with this change.

### PR DESCRIPTION
**Pull request by makkoncept**
## Fixes

Fixes #105 by @panchovm 

## Description

Add three icons
1. Adjust
2. Bookmark regular
3. Bookmark solid

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by makkoncept** [comment]
Hi @panchovm, @zackkrida. I have looked at https://github.com/FortAwesome/Font-Awesome/tree/master/svgs/solid and couldn't find the "bookmark regular" SVG file. There is only "bookmark solid" which is named `bookmark.svg`.

Edit: I found it. It's obviously in `svgs/regular` and not in `svgs/solid`. My bad :sweat_smile:

**Comment by zackkrida** [comment]
@makkoncept this should be the 'empty' (outline only) icon:
 https://www.github.com/FortAwesome/Font-Awesome/tree/master/svgs%2Fregular%2Fbookmark.svg

File: src/styles/vocabulary_icons.scss [hunk 1]
```diff
@@ -60,6 +60,9 @@ $icons: (
   "question-circle": "\ea0c",
   "plus": "\ea0d",
   "minus": "\ea0e",
+  "adjust": "\ea0f",
+  "bookmark-regular": "\eb00",
```
Comment (dhruvkb) [inline src/styles/vocabulary_icons.scss]: `ea0f` is followed by `ea10`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please fix the small calculation error.

**Review by hugosolar — APPROVED** [review]
Beside Dhruv's comment I don't have any other concern, it looks good to me

**Review by zackkrida — APPROVED** [review]
See @Dhruvi16's requested fix.

**Comment by zackkrida** [comment]
looks good @makkoncept thanks for fixing the unicode sequence

**Pull request merged**